code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1983,6 +1988,14 @@ var/global/list/organ_damage_overlays = list(
 				visible_message("<span class='warning'>The bucket's content spills on [src]</span>")
 				spawn(5) B.reagents.clear_reagents()
 
+/mob/living/carbon/human/proc/handle_heartattack()
+	if(!heart_attack)
+		return
+	else
+		losebreath += 5
+		adjustOxyLoss(5)
+		adjustBrainLoss(10) // without oxygin brain dont evoluiate

```

Comment:
Real heart attacks give brain damage in a few minutes, and this is also snowflake code that should fall under the oxygen intake code for humans.